<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioOptionsDlg</class>
 <widget class="QDialog" name="AudioOptionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="dlgAudio">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkAudioOnly">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>251</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Keep video file after conversion</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Best</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>MP3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>AAC</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>M4A</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>VORBIS</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>OPUS</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>VORBIS</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>WAV</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="chkAudioExtract">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract Audio</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Format</string>
   </property>
  </widget>
  <widget class="QSlider" name="sldQuality">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Quality</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>High</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>dlgAudio</zorder>
  <zorder>chkAudioOnly</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>radioButton_4</zorder>
  <zorder>radioButton_5</zorder>
  <zorder>radioButton_6</zorder>
  <zorder>radioButton_8</zorder>
  <zorder>radioButton_7</zorder>
  <zorder>chkAudioExtract</zorder>
  <zorder>sldQuality</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dlgAudio</sender>
   <signal>accepted()</signal>
   <receiver>AudioOptionsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dlgAudio</sender>
   <signal>rejected()</signal>
   <receiver>AudioOptionsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
